### ISSUE TITLE
Write data with keep format of cell

### ISSUE THREAD

**Issue by zkday** [dtjohnson/xlsx-populate#17]
I have template file with some settings at the format for cells. (e.g the cell "A1" have been setting as "Currency") but when write data with:
`var workbook = Workbook.fromFileSync("./Template1.xlsx");
workbook.getSheet("Sheet1").getCell(1,1).setValue(1000);
workbook.toFileSync("./out.xlsx");
`
After success the format at cell "A1" of out.xlsx have been changed to "General". How can I keep the format of cell as Template file? 

Thanx.

**Comment by dtjohnson** [comment]
Could you share the workbook?

**Comment by zkday** [comment]
[Template.xlsx](https://github.com/dtjohnson/xlsx-populate/files/752999/Template.xlsx)

This is the workbook. 
Thanx.

**Comment by dtjohnson** [comment]
Thanks. So this seems to be yet another oddity of the Office Open XML spec. It looks like there is indeed a style attached to the column. xlsx-populate also dutifully adds the cell value. But it seems that the style on the column doesn't actually do anything. In order for the cell to get the style applied, xlsx-populate should copy the style reference from the column to the cell. It doesn't currently do that. We'll try to get it addressed in a future release.

The workaround for now is to style the cells in the column rather than the entire column. (If you use the same template you'll need to remove the style from the column first.)

**Comment by zkday** [comment]
Thanks for feedback. Please let me know if i can help!

**Comment by dtjohnson** [comment]
I just released v1.0.0. It now properly handles column styles. The API has changed though. Now your example should be:
```js
const XlsxPopulate = require("xlsx-populate");
XlsxPopulate.fromFileAsync("./Template1.xlsx")
    .then(workbook => {
        workbook.sheet("Sheet1").cell(1, 1).value(1000);
        return workbook.toFileAsync("./out.xlsx");
    });
```
